DNN Platform/Library/Services/Log/EventLog/LogTypeConfigInfo.cs

```diff
@@ -66,7 +69,17 @@ public DateTime StartDateTime
 
         public bool EmailNotificationIsActive { get; set; }
 
-        public string MailFromAddress { get; set; }
+        public string MailFromAddress
```

Comment:
Actually If senderadress is empty then email was not getting shoot. So set it with the current portal and also checked for web application and Scheduler process as well.